rollup.config.ts

```diff
@@ -43,6 +43,7 @@ export function handlebarsPlugin(): Plugin {
                         exactArray: true,
                         ifdef: true,
                         ifNotNullNotUndefined: true,
+                        ifNotUndefined: true,
```

Comment:
I think you can get the same outcome if you do `#notEquals value undefined`, have you tried that?